### PR TITLE
added a file with tasks 3.1 - 3.7

### PR DESCRIPTION
**Pull request by katyasolovii**
work in the lesson from 3.1 to 3.7

### CODE REVIEW

File: t3.c [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+    }
+}
+
+void task7_b(){
+
+    double a, b, c;
+    printf("a, b, c: ");
+    scanf("%lf %lf %lf", &a, &b, &c);
+
+    if (a == 0 && b == 0){
```
Comment (mecchmatProjects) [inline t3.c]: a==0 is a BAD comparison for doubles

Comment (katyasolovii) [reply t3.c]: I fixed this task. Please, check it out

Comment (mecchmatProjects) [reply t3.c]: No changes I see

**Review by mecchmatProjects — APPROVED** [review]
Sorry, see the changes

**Pull request merged**